com.unity.render-pipelines.universal/Documentation~/configuring-universalrp-for-use.md

```diff
@@ -14,7 +14,7 @@ The [Universal Render Pipeline Asset](universalrp-asset.md) controls the [global
 To create a Universal Render Pipeline Asset:
 
 1. In the Editor, go to the Project window.
-2. Right-click in the Project window, and select  __Create > Rendering > Universal Render Pipeline > Pipeline Asset__. Alternatively, navigate to the menu bar in top, and click __Assets > Create > Rendering > Universal Render Pipeline > Pipeline Asset__.
+2. Right-click in the Project window, and select  __Create > Rendering > URP Pipeline Asset__. Alternatively, navigate to the menu bar in top, and click __Assets > Create > Rendering > Universal Render Pipeline > Pipeline Asset__.
```

Comment:
URP Pipeline Asset shouldn't this be URP asset?

Reply:
done, thanks :)